app/views/posts/show.html.slim

```diff
@@ -1,35 +1,69 @@
 h1.display-4.text-center.font-weight-bold.mt-5 Selected Post
-.card.mt-5
-  .card-body.bg-light
+.card.bg-secondary.mt-5
+  .card-body
     h4.card-title.font-weight-bold.text-center
       = @post.title
-    .row.mt-3
-      .col-lg-4
-        h5.card-title.font-weight-bold
+
+    .div.d-flex.justify-content-between.mt-3
+      .div
+        h5.card-title.font-weight-bold.float-left
           | Requester:
+        h5.card-title.float-left.ml-2.text-light
           = @post.requester
 
-      .col-lg-4
-        h5.card-title.font-weight-bold
+      .div
+        h5.card-title.font-weight-bold.float-left
           | Telephone Number:
+        h5.card-title.float-left.ml-2.text-light
           = @post.telephone
 
-      .col-lg-4
-        h5.card-title.font-weight-bold
+      .div
+        h5.card-title.font-weight-bold.float-left
           | Required BloodType:
+        h5.card-title.float-left.ml-2.text-light
           = @post.bloodtype
 
-    p.card-text.lead
+    p.card-text.lead.text-light
       = @post.description
 
-    = link_to 'Delete', post_path, method: :delete, data: { confirm: "Are you sure?" }, class: 'btn btn-sm btn-danger float-right'
+    h5.card-title.font-weight-bold.float-left
+      | Donator:
+    - if !@post.donator.nil?
```

Comment:
- if @post.donator.empty? or  - if !@post.donator.blank?

Reply:
Noted. Any specific reason for not using "if !@post.donator.nil?"